### PR TITLE
Add workflow and RPC for Optimism Kovan

### PR DESCRIPTION
**Pull request by drcpu-github**

### CODE REVIEW

File: .github/workflows/cron.testnet.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Install Dependencies
+        id: installDependencies
+        uses: bahmutov/npm-install@v1
+        with:
+          install-command: yarn
+      - name: Install PoolTogether V4 CLI
+        run: |
+          npm install -g @pooltogether/v4-cli@0.1.6
```
Comment (kamescg) [inline .github/workflows/cron.testnet.yml:37]: It's not ideal, but we also have to update the v4-cli version dependency here.

https://www.npmjs.com/package/@pooltogether/v4-cli

Current version is [0.1.7-beta.2](https://www.npmjs.com/package/@pooltogether/v4-cli/v/0.1.7-beta.2)

**Review by kamescg — CHANGES_REQUESTED** [review]

File: .github/workflows/cron.testnet.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          echo "$(npm -g bin)" >> $GITHUB_PATH
+      # -------------------------------------------------------- #
+      # Optimism Kovan (USDC)
+      # -------------------------------------------------------- #
+      - name: Optimism Kovan Draw Prizes
+        id: drawPrizesOptimismKovan
+        uses: maddog2050/executable-wrapper@v1
+        continue-on-error: true
+        with:
+          run_command: ptv4 compute poolPrizes -o ./api/testnet/prizes -t 0x2540Be4EE62A5E01F27491044AeBb9C2c95A550A -c 69
```
Comment (kamescg) [inline .github/workflows/cron.testnet.yml:47]: Update `./api/testnet/prizes` to `./api/prizes`

I made the wrong call in the bounty specification regarding this update.

**Review by kamescg — CHANGES_REQUESTED** [review]

File: .github/workflows/cron.testnet.yml [hunk 3]
```diff
@@ -0,0 +1,84 @@
+name: CronJob
+on:
+  schedule:
+    - cron: "0 20,21 * * *" # everyday at 8pm and 9 pm UTC
```
Comment (kamescg) [inline .github/workflows/cron.testnet.yml:4]: The cron job should match the frequency of the Rinkeby DrawBeacon draw rate.

https://rinkeby.etherscan.io/address/0x7025879c052bbac44fb24600087fe30a0f32edfe

Comment (drcpu-github) [reply .github/workflows/cron.testnet.yml:4]: Looking at the [contract](https://rinkeby.etherscan.io/address/0x422193556f47b17589325c50af478c724be976e2#readContract), I guess that's every half hour?

**Pull request merged**